run.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
+#!/usr/bin/env bash
+# -*- coding: utf-8; mode: bash  -*-
+
```

Comment:
Sorry, that doesn't really belong in here. I realized that too late .. the patch was already pushed remote :-o